gatsby-config.js

```diff
@@ -104,6 +108,28 @@ module.exports = {
             path: '/server/reference/'
           }
         ]
+      },
+      {
+        title: 'Recipes',
+        path: '/recipes/',
```

Comment:
What happens to the existing recipes?

Reply:
The existing recipes are now on the main landing page of recipes. The new recipes are on the side navigation bar.  
<img width="1726" alt="Screenshot 2023-05-25 at 12 41 48 AM" src="https://github.com/AdobeDocs/uxp-indesign/assets/42615528/0d41eebb-a5ff-4630-b8d4-3553d8232045">